gpu-operator/platform-support.rst

```diff
@@ -399,6 +405,27 @@ The GPU Operator has been validated in the following scenarios:
          -
 
 
+Supported Precompiled Drivers
+-----------------------------
+
+The GPU Operator has been validated with the following precomplied drivers.
+See the :doc:`precompiled-drivers` page for more on using precompiled drivers.
+
++----------------------------+------------------------+----------------+---------------------+
+| Operating System           | Kernal Flavor          | Kernal Version | CUDA Driver Branch  |
++============================+========================+================+=====================+
+| Ubuntu 22.04               | Generic                | 5.15           | R535, R550          |
```

Comment:
Don't we need to add `Ubuntu 20.04` as well?

Reply:
We don't publish precompiled containers for 20.04.

Reply:
Ah of course, silly me